common/fs/src/tail.rs

```diff
@@ -129,20 +127,24 @@ impl Tailer {
                             if !paths.is_empty() {
                                 if let Entry::File {
                                     ref mut data,
-                                    file_handle,
                                     ..
                                 } = entry.borrow_mut().deref_mut()
                                 {
                                     info!("added {:?}", paths[0]);
-                                    *data = 0;
-                                    if let Some(mut lines) = Tailer::tail(file_handle, &paths, data) {
-                                        final_lines.append(&mut lines);
-                                    }
+                                    data.seek(0).unwrap_or_else(|e| error!("error seeking {:?}", e));
```

Comment:
What's the reason to seek to 0?

`BufReader`'s `seek()` method will discard the internal buffer even if the offsets match.

Reply:
Yeh, I think this is a no-op as it's only used for New files. This is just a leftover from the previous implementation, I'll remove it.